src/lando/jinja.py

```diff
@@ -314,7 +314,11 @@ def bug_url(text: str) -> str:
     )
 
 
-def revision_url(revision_id: int | str, diff_id: Optional[str] = None) -> str:
+def pull_request_link(repo: Repo, revision: Revision) -> str:
+    return f"{repo.url.removesuffix('.git')}/pull/{ revision.pull_number }"
+
+
+def revision_url(revision_id: int | str, diff_id: str | None = None) -> str:
```

Comment:
Might be time to give this a more descriptive name, maybe `phabricator_revision_url`.